molgenis-compute-core/src/main/resources/templates/localhost/footer.ftl

```diff
@@ -1,9 +1,22 @@
+<#noparse>#
+## Start of footer for backend 'local'.
 #
-## General footer
-#
 
-# Show that we successfully finished. If the .finished file exists, then this step will be skipped when you resubmit your workflow 
-touch $ENVIRONMENT_DIR/${taskId}.sh.finished
+if [ -d "${MC_tmpFolder:-}" ]; then
+	echo -n "INFO: Removing MC_tmpFolder ${MC_tmpFolder} ..."
+	rm -rf "${MC_tmpFolder}"
+	echo 'done.'
+fi
+
+tS=${SECONDS:-0}
+tM=$((SECONDS / 60 ))
+tH=$((SECONDS / 3600))
+echo "On $(date +"%Y-%m-%d %T") ${MC_jobScript} finished successfully after ${tM} minutes." >> molgenis.bookkeeping.log
+printf '%s:\t%d seconds\t%d minutes\t%d hours\n' "${MC_jobScript}" "${tS}" "${tM}" "${tH}" >> molgenis.bookkeeping.walltime
+
+mv "${MC_jobScript}.started" "${MC_jobScript}.finished"
```

Comment:
In submit.ftl, you use "mv molgenis.submit.{started,finished}". To make it consistent, use it here too? 
"mv ${MC_jobScript}.{started,finished}"